### PR TITLE
Additional options to mapping-sheet

### PR DESCRIPTION
**Pull request by romifz**
The following options has been added to mapping-sheet:

* Generate from URL: both through the form and through a GET request
using the "source" parameter.
* Upload a file: using a file upload form field.
* Extensions from release schema: creates a patched release schema with
one or more Extensions, both through form fields or providing the URLs
with the "extension" parameter in a GET request.

ref #12

### CODE REVIEW

File: default/mapping_sheet.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+def get_mapping_sheet_from_uploaded_file(uploaded):
+    schema = jsonref.loads(uploaded.read())
+
+    return _get_mapping_sheet(schema)
+
+
+def get_extended_mapping_sheet(extensions):
+    cove_config = LibCoveOCDSConfig()
+    schema_ocds = SchemaOCDS('1.1', extensions, lib_cove_ocds_config=cove_config)
```
Comment (jpmckinney) [inline default/mapping_sheet.py]: I suggest instead using the ProfileBuilder from extension_registry.py

It has the advantage over SchemaOCDS of being [documented](https://ocdsextensionregistry.readthedocs.io/en/latest/api/profile_builder.html), having a stable API, and being used in OCDS Kit, [see example usage](https://github.com/open-contracting/ocdskit/blob/master/ocdskit/cli/commands/mapping_sheet.py#L29-L31)

Comment (romifz) [reply default/mapping_sheet.py]: Thanks! I will update this.

File: default/forms.py [hunk 2]
```diff
@@ -1,5 +1,51 @@
 from django import forms
+from django.utils.translation import gettext as _
+from .mapping_sheet import get_standard_tags
+
+TYPE_CHOICES = (('select', 'Select a Schema'),
```
Comment (jpmckinney) [inline default/forms.py]: Should these be translated by gettext?

File: default/mapping_sheet.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+from io import StringIO
+from django.http.response import HttpResponse
+from ocdskit.mapping_sheet import mapping_sheet as mapping_sheet_method
+from django.core.cache import cache
+from ocdsextensionregistry import ProfileBuilder
+from ocdsmerge.merge import get_tags
+
+CACHE_KEY = 'git_tags'
+CACHE_TIMEOUT = 3600
```
Comment (jpmckinney) [inline default/mapping_sheet.py]: This constant is not used anywhere?

File: default/forms.py (hunk 2 above)
Comment (romifz) [reply default/forms.py]: These are translated in the templates.

File: default/mapping_sheet.py (hunk 3 above)
Comment (romifz) [reply default/mapping_sheet.py]: It should be! Thanks for noticing.

**Pull request merged**